### PR TITLE
corrected a vovel error

### PR DESCRIPTION
**Pull request by yasoob**

### CODE REVIEW

File: loklak.py [hunk 1]
```diff
@@ -242,5 +242,5 @@ def account(self, name=None, action=None, data=None):
         else:
             return_to_user = {}
             return_to_user['error'] = ('No Query string has been given'
-                                       ' to run an query for account')
+                                       ' to run a query for an account')
```
Comment (sudheesh001) [inline loklak.py]: please correct this as `No Query string has been given to query for an account`

**Comment by mariobehling** [comment]
@sudheesh001 Please review and resolve merge conflicts to get this PR finished, if possible.

**Comment by sudheesh001** [comment]
@yasoob Are you still working on this, Could you please rebase ?

Test release : 24 August 2016

**Comment by mariobehling** [comment]
@yasoob If this PR is still valid from your side, please resubmit. Thank you.

**Pull request closed without merging**